TestingApp/Info.plist

```diff
@@ -24,6 +24,8 @@
 	<string>This app needs camera for audio/video calls and photo sending</string>
 	<key>NSMicrophoneUsageDescription</key>
 	<string>This app needs microphone for audio/video calls</string>
+	<key>NSPhotoLibraryAddUsageDescription</key>
+	<string>Save attachments</string>
```

Comment:
Maybe we should make a little bit more verbose? e.g. "This app needs access to photo library to save attachments" or smth like this.

Reply:
Ah, sorry, you are right. Didn't notice that it's from the testing app.